delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ResolversRoutesSpec.scala

```diff
@@ -52,20 +52,22 @@ class ResolversRoutesSpec
   private val asAlice = addCredentials(OAuth2BearerToken(alice.subject))
   private val asBob   = addCredentials(OAuth2BearerToken(bob.subject))
 
-  private val org      = Label.unsafe("org")
-  private val am       = ApiMappings(Map("nxv" -> nxv.base, "Person" -> schema.Person))
-  private val projBase = nxv.base
-  private val project  =
+  private val org                = Label.unsafe("org")
+  private val defaultApiMappings = ApiMappings("_" -> schemas.resources, "resource" -> schemas.resources)
+  private val am                 = ApiMappings("nxv" -> nxv.base, "Person" -> schema.Person)
+  private val projBase           = nxv.base
+  private val project            =
     ProjectGen.project("org", "project", uuid = uuid, orgUuid = uuid, base = projBase, mappings = am)
-  private val project2 =
+  private val project2           =
     ProjectGen.project("org", "project2", uuid = uuid, orgUuid = uuid, base = projBase, mappings = am)
 
   private val (_, projects) = {
     implicit val subject: Subject = Identity.Anonymous
     ProjectSetup
       .init(
         orgsToCreate = List(org),
-        projectsToCreate = List(project, project2)
+        projectsToCreate = List(project, project2),
+        defaultApiMappings = defaultApiMappings
```

Comment:
And put Resources.mappings as a default value here as it seems that is what we use during the tests when it does not really matter